R/ui.R

```diff
@@ -90,7 +86,7 @@ HarmonyMatrix <- function(
     if (hasArg(do_pca) || hasArg(npcs)) {
         stop('Error: Function parameters do_pca and npcs have been removed in newer versions of harmony. Please remove any of the do_pca or npcs parameters and pass to harmony cell_embeddings directly')
     }
-
+    opt_args <- list(...)
```

Comment:
It seems that lambda is already set in the `lambda` here if it is passed as a parameter. What is the point of this?

Reply:
I thought `list(...)` is the standard  way to parse ... arguments